packages/input-tel-dropdown/test-suites/LionInputTelDropdown.suite.js

```diff
@@ -71,6 +71,33 @@ export function runInputTelDropdownSuite({ klass } = { klass: LionInputTelDropdo
       await PhoneUtilManager.loadComplete;
     });
 
+    it('syncs value of dropdown on init if input has no value', async () => {
+      const el = await fixture(html` <${tag}></${tag}> `);
+      expect(el.activeRegion).to.equal('GB');
+      expect(el.value).to.equal('+44');
+      expect(getDropdownValue(/** @type {DropdownElement} */ (el.refs.dropdown.value))).to.equal(
+        'GB',
+      );
+    });
+
+    it('syncs value of dropdown on reset if input has no value', async () => {
+      const el = await fixture(html` <${tag}></${tag}> `);
+      el.modelValue = '+31612345678';
+      await el.updateComplete;
+      expect(el.activeRegion).to.equal('NL');
+      el.reset();
+      await el.updateComplete;
+      expect(el.activeRegion).to.equal('GB');
+      expect(el.value).to.equal('+44');
+    });
+
+    it('syncs value of dropdown on init does not influence interaction states', async () => {
+      const el = await fixture(html` <${tag}></${tag}> `);
+      // TODO find out why its get dirty again
```

Comment:
Via  `_initSetModelValueBasedOnDropdown() {` it is set to `false`, but somewhere it gets dirty again. Which is another bug, not part of this fix.